tests/optim/test_oss.py

```diff
@@ -234,6 +235,7 @@ def closure():
     optimizer.load_state_dict(optimizer_state_dict)
 
 
+@skip_if_not_enough_gpu
```

Comment:
This should be a separate change. Each change should have a single thesis.

Correct fix in this case might be to set world_size to number of cuda devices or 3.

Reply:
You are right. Thanks for the suggestion. Let me move this out.